cg/cli/upload/scout.py

```diff
@@ -252,7 +239,7 @@ def upload_multiqc_to_scout(context: CGConfig, case_id: str, dry_run: bool) -> N
 def get_upload_api(case: Family, cg_config: CGConfig) -> UploadAPI:
     """Return the upload API based on the data analysis type"""
 
-    analysis_apis: Dict[Pipeline, UploadAPI] = {
```

Comment:
Do you not like the verbose hint?

Reply:
The verbose hint was incorrect - it was in matter of fact not returning the metaclass but a long union of the different upload apis which I found more complex